manifests/master/config.pp

```diff
@@ -92,9 +94,10 @@
   # because it polls it automatically.
   if ($manage_report_processor) {
     class { 'puppetdb::master::report_processor':
-      puppet_conf => $puppet_conf,
-      enable      => $enable_reports,
-      require     => $strict_validation ? {
+      puppet_conf         => $puppet_conf,
+      puppet_conf_section => $puppet_conf_section,
+      enable              => $enable_reports,
+      require             => $strict_validation ? {
         true    => Puppetdb_conn_validator['puppetdb_conn'],
         default => Package[$terminus_package],
```

Comment:
@ajroetker ues you still need the terminus on every node https://docs.puppetlabs.com/puppetdb/2.2/connect_puppet_apply.html#step-2-install-terminus-plugins-on-every-puppet-node